app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -1680,6 +1684,38 @@ class PluginApi {
   addTopicsSectionLink(arg) {
     addSectionLink(arg);
   }
+
+  /**
+   * Decorate inline bbcode in to-markdown function.
```

Comment:
I think we're referring to the `toMarkdown` function in the `to-markdown` file? Also, I'm not sure if this is tied to `bbcode` because I can technically insert an `prefix` or `suffix` which are not bbcode?